docs/install/local/README.md

```diff
@@ -109,13 +105,27 @@ If you select this option, it will ask if you want to run the service as the
 current user, or create a new `flowforge` user. If you choose to create the
 user, it will also change the ownership of the FlowForge directory to that user.
 
-### Configuring FlowForge
+## Configuring FlowForge (optional)
```

Comment:
```suggestion
## Configuring FlowForge
```

I think this section contains important information (the location of the config file) that a user should be encouraged to read. I do not believe we should mark it optional.

I suggest changing the text after the file locations to say:

> The default configuration file already contains everything you need to get started with FlowForge.
> 
> To customise your installation, see the [configuration guide](../configuration.md).

However, we *do* have the issue with the `host` property - if the user leaves it as the default, and create a project, they will be stuck when they realise they want it to be more widely accessible as we don't support changing the `host` value after projects are made.

Reply:
NP, will make the necessary adjustments shortly.

Reply:
@knolleary @hardillb 
Regarding the `host` comment - I see 3 options...

1. Dont mention it (not ideal)

1. Mention it in line (in the README)...

> The default configuration file already contains everything you need to get started with FlowForge.
> 
> To customise your installation, see the [configuration guide](https://github.com/flowforge/flowforge/configuration.md) in particular, not setting the `host` value will mean that projects will not be accessible outside of the FlowForge server and we currently do not support changing the `host` value for existing projects.

1. Mention it in the [configuration guide](https://github.com/flowforge/flowforge/configuration.md).

> 
> Option | Description
> -------|------------
> `host` | The address to serve the web ui on. This defaults to `localhost` which means the ui will only be available when browsing from the same server that is running the platform. To make it accessible to other devices on the network, set it to `0.0.0.0`.  NOTE: We currently do not support changing the `host` value for existing projects.

--- 

please comment :)

Reply:
But if `host` is changed `base_url` also needs changing as this is used to find the oauth endpoints when signing into a project and defaults to `http://locahost:3000`

Reply:
> But if host is changed base_url also needs changing as this is used to find the oauth endpoints when signing into a project and defaults to http://locahost:3000

:doh:

Do you have a suggestion pls?

Reply:
Add a note to the `host` entry to say that if changed to update `base_url` to `http://[ip-address-of-host]:3000`

Reply:
ok, looks like this...
![image](https://user-images.githubusercontent.com/44235289/203048048-4cbda379-f183-4bd3-b588-7409833c31e5.png)

Gonna push that in 3... 2... 1...

Reply:
I believe this is now resolved/outdated. Will mark as resolved for sake of clarity & re-request reviews shortly after some housekeeping